tests/corrupted_stamp_test.cpp

```diff
@@ -350,25 +366,25 @@ void test_read_corrupted_stamp(ioremap::elliptics::newapi::session &s, const dne
 		// Different offests and data size requests.
 		//
 		{
-			auto async = s.read_data(write_result.key, 11, 101);
+			auto async = s.read_data(write_result.key, 11, 81);
```

Comment:
Why was it changed?

Reply:
`sizeof(dc)` = 96, `sizeof(ehdr)` = ~40.

96 + 40 < 11 + 101, so the test was working.

But now we have the case of `dc` only. Tests with old values became broken.

Reply:
`sizeof(ehdr) = 48`. Please add a comment why `81` is used here.